### PR TITLE
Add XML formatter

### PR DESCRIPTION
**Pull request by dmitriy-strukov**

### CODE REVIEW

**Comment by dmitriy-strukov** [comment]
@davydovanton Hello )), can you please review?

**Comment by davydovanton** [comment]
@dmitriy-strukov could you fix CI build?

File: lib/hanami/events/formatter/xml.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      # @since 0.1.0
+      #
+      # @api private
+      class Xml
+        def initialize(events_meta)
+          # Accepts meta data of subscribed events
+          #
+          # @param events_meta [Hash] events meta data
+          #
+          # @since 0.1.0
```
Comment (artofhuman) [inline lib/hanami/events/formatter/xml.rb]: hm, i think the documentation should be above method definition

**Comment by dmitriy-strukov** [comment]
@davydovanton done, thank you for review!

**Comment by artofhuman** [comment]
👍

File: lib/hanami/events/formatter/xml.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        # @since 0.1.0
+        def initialize(events_meta)
+          @events_meta = events_meta
+        end
+
+        # Returns events meta data in xml format
+        #
+        # @since 0.1.0
+        def format
+          XmlSimple.xml_out({ events: @events_meta })
```
Comment (smakagon) [inline lib/hanami/events/formatter/xml.rb]: Hi, @dmitriy-strukov. Thanks for the PR!
Curly braces could be omitted here.

File: Gemfile [hunk 3]
```diff
@@ -11,3 +11,4 @@ end
 
 gem 'simplecov', require: false, group: :test
 gem 'mutant-rspec'
+gem 'xml-simple'
```
Comment (davydovanton) [inline Gemfile]: Could you add this dependency to gemspec?
https://github.com/hanami/events/blob/master/hanami-events.gemspec#L28

File: spec/unit/hanami/events/formatter/xml_spec.rb [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+      { title: 'user.*' },
+      { title: 'user.created' },
+      { title: 'post.*' },
+      { title: 'post.created' },
+    ]
+  end
+
+  describe '#format' do
+    it 'formats events data' do
+      expect(formatter.format).to eq(XmlSimple.xml_out({ events: data }))
```
Comment (smakagon) [inline spec/unit/hanami/events/formatter/xml_spec.rb]: To keep things consistent, could you please remove curly braces here too, please.

**Pull request merged**